.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Is this a nix specific file?  Do I have to worry about any weird errors when working in the repo when I don't use nix?

Reply:
Yup, it should load the nix flake if you have nix and direnv installed on your machine, but I think there shouldnt be any issues on other machines without nix installed